moped-editor/src/views/projects/projectView/ProjectPhase/form.js

```diff
@@ -0,0 +1,47 @@
+import * as yup from "yup";
+
+export const phaseValidationSchema = yup.object().shape({
+  phase_id: yup
+    .number("Phase is required")
+    .nullable()
+    .required("Phase is required"),
+  subphase_id: yup.number().nullable().optional(),
+  phase_start: yup.string().nullable().optional(),
+  phase_end: yup.string().nullable().optional(),
+  is_current_phase: yup.boolean(),
+  is_phase_start_confirmed: yup.boolean(),
+  is_phase_end_confirmed: yup.boolean(),
+  description: yup
+    .string()
+    .max(500, "Must be less than 500 characters")
```

Comment:
i think we've talked about this before, but max string length is not something we enforce at the db level, and we probably should 🙃